docs/pages/includes/config-reference/database-config.yaml

```diff
@@ -4,15 +4,16 @@ db_service:
 
   # Matchers for database resources created with "tctl create" command or by the
   # discovery service.
-  #
-  # All database resources have a predefined "teleport.dev/origin" label with
-  # one of the following values:
-  # "dynamic": database resources created with "tctl create" command
-  # "cloud": database resources created by the discovery service
-  # "config": database resources defined in the "databases" array below
   resources:
+    # See "Database labels reference" (https://goteleport.com/docs/database-access/reference/labels/)
```

Comment:
Do relative links work in partials or no?

Reply:
If you're asking about links in YAML snippet comments, we don't have a way to support this yet. 

This would pose the same technical challenge as supporting the `Var` component in non-`code` snippets (i.e., manipulating the HTML AST created by our syntax highlighting plugin), but I don't think it's out of the question for the future.